### PR TITLE
Initial configuration of prestashop, include database backup

### PR DESCRIPTION
**Pull request by oenea**

### CODE REVIEW

File: src/backup.db
Comment (Nasus20202) [inline src/backup.db:1]: Could you please add the scripts that you used to export/import the configuration of the Prestashop?

**Review by Nasus20202 — CHANGES_REQUESTED** [review]
It would be great to have a README describing how to import/backup the configuration. Also, make sure to run the seeder script before exporting the final configurations. Check if it works both on MariaDB and MySQL and add such note to the README.

**Comment by Nasus20202** [comment]
FYI: after loading the backup by hand, I got this error: `Fatal error: require_once(): Failed opening required '/app/prestashop/var/cache/dev/appParameters.php' (include_path='/app/prestashop/vendor/pear/pear_exception:/app/prestashop/vendor/pear/console_getopt:/app/prestashop/vendor/pear/pear-core-minimal/src:/app/prestashop/vendor/pear/archive_tar:.:/usr/local/lib/php') in /app/prestashop/config/bootstrap.php on line 87`. Could be an issue with how I imported the backup.

**Comment by fildaw** [comment]
> FYI: after loading the backup by hand, I got this error: `Fatal error: require_once(): Failed opening required '/app/prestashop/var/cache/dev/appParameters.php' (include_path='/app/prestashop/vendor/pear/pear_exception:/app/prestashop/vendor/pear/console_getopt:/app/prestashop/vendor/pear/pear-core-minimal/src:/app/prestashop/vendor/pear/archive_tar:.:/usr/local/lib/php') in /app/prestashop/config/bootstrap.php on line 87`. Could be an issue with how I imported the backup.

Having the same error, imported `backup.sql` file using command `mariadb -u root -p < backup.db` in fresh container

**Comment by fildaw** [comment]
> > FYI: after loading the backup by hand, I got this error: `Fatal error: require_once(): Failed opening required '/app/prestashop/var/cache/dev/appParameters.php' (include_path='/app/prestashop/vendor/pear/pear_exception:/app/prestashop/vendor/pear/console_getopt:/app/prestashop/vendor/pear/pear-core-minimal/src:/app/prestashop/vendor/pear/archive_tar:.:/usr/local/lib/php') in /app/prestashop/config/bootstrap.php on line 87`. Could be an issue with how I imported the backup.
> 
> Having the same error, imported `backup.sql` file using command `mariadb -u root -p < backup.db` in fresh container

This error does not show if I preserve all prestashop files, perhaps we should not include all cache files in .gitignore? Maybe it is worth taking a look at official (I think) [gitignore](https://github.com/github/gitignore/blob/main/Prestashop.gitignore) for Prestashop.

**Pull request closed without merging**